diff --git a/desktop_app/__main__.py b/desktop_app/__main__.py
index 6a2ddeb..6ce5825 100644
--- a/desktop_app/__main__.py
+++ b/desktop_app/__main__.py
@@ -27,6 +27,15 @@ def main():
             defaults to the Start menu on Windows, and to ~/.local/share/applications on
             Linux.""",
     )
+
+    parser.add_argument(
+        '--no-fix-entry-points',
+        dest='no_fix_entry_points',
+        default=False,
+        action='store_true',
+        help="If set, entry points will not be fixed for conda-environments "
+        "during the install action."
+    )
     parser.add_argument(
         '-q',
         '--quiet',
@@ -38,7 +47,8 @@ def main():
     args = parser.parse_args()
     if args.action == 'install':
         for module in args.module:
-            install(module, path=args.path, verbose=not args.quiet)
+            install(module, path=args.path, verbose=not args.quiet,
+                    no_fix_entry_points=args.no_fix_entry_points)
     if args.action == 'uninstall':
         for module in args.module:
             uninstall(module, path=args.path, verbose=not args.quiet)
diff --git a/desktop_app/shell.py b/desktop_app/shell.py
index f53e1a8..1f9fec1 100644
--- a/desktop_app/shell.py
+++ b/desktop_app/shell.py
@@ -207,7 +207,7 @@ def _launcher_script_symlink_path(config):
             return config.launcher_script_path.parent / config.appid
 
 
-def install(module_name, path=None, verbose=False):
+def install(module_name, path=None, verbose=False, no_fix_entry_points=False):
     """Add a shortcut to launch the app for the given module to the system menus, i.e
     the start menu on Windows, ~/.local/share/applications on Linux, and TODO on macOS.
     If path is given, the shortcut will be created in the given directory instead. On
@@ -257,7 +257,8 @@ def install(module_name, path=None, verbose=False):
             # does not support gui_scripts, so those entry_points may have been
             # installed as console_scripts:
             dist = get_distribution_of_module(module_name)
-            fix_entry_points(dist)
+            if not no_fix_entry_points:
+                fix_entry_points(dist)
     elif LINUX:
         symlink_path = _launcher_script_symlink_path(config)
         create_desktop_file(
