From ae57179538c1cfe743765ba06d1427d6eeeef070 Mon Sep 17 00:00:00 2001
From: chrisjbillington <chrisjbillington@gmail.com>
Date: Thu, 4 Mar 2021 07:52:26 +1100
Subject: [PATCH] Fix #15, allow launching single-file modules

---
 desktop_app/launcher.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/desktop_app/launcher.py b/desktop_app/launcher.py
index 8ce7272..76439be 100644
--- a/desktop_app/launcher.py
+++ b/desktop_app/launcher.py
@@ -55,6 +55,8 @@ def entry_point():
     script_path = Path(package_directory, *module_name.split('.')[1:])
     if script_path.is_dir():
         script_path /= '__main__.py'
+    else:
+        script_path = script_path.with_suffix('.py')
 
     popen_kwargs = {}
 
